### ISSUE TITLE
Navbar: keyboard navigation doesn't work for nested components

### ISSUE THREAD

**Issue by tishoyanchev** [Infineon/infineon-design-system-stencil#1918]
### Detailed description of how to reproduce the problem

N/A

### Upload a screenshot

N/A

### Describe the development framework

N/A

### Expected behavior

N/A

**Comment by tishoyanchev** [comment]
This issue will be worked at in the already opened PR
